testing/php7-apcu/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
```

Comment:
better to let @andypost claim maintainership (if he want to move it to community)

PS i don't think we have a standard for local based variables, until we do so it shouldn't matter what to use.